### ISSUE TITLE
Roads are too prominent compared to landcover and rivers at mid and low zoom levels

### ISSUE THREAD

**Issue by jeisenbe** [gravitystorm/openstreetmap-carto#3935]
## Expected behavior
- Highways should be visible but not overly prominent at mid and low zoom levels
- Landcover features should be balance the linear features
- Rivers and railways should be visually balanced with primary and trunk highways
- These visual features should encourage mappers to add waterways, water features, landcover and landuse features, not only roads

## Actual behavior
- Because the landcover layer is lightened by 0.6, roads are much more prominent than landcover at z10 and below. 
- At z11 and z12 the 0.4 and 0.5 lightening of landcover still makes roads more prominent, even the tertiary roads and unclassified/residential roads rendered as think gray lines
- Railways and rivers are too prominent compared to landcover at z12 and below, but are less prominent than motorways and trunk roads
- This suggests that this style is focused on roads, especially motorroads, at mid and low-mid zoom levels.

### Schwerin
https://www.openstreetmap.org/#map=10/53.70615/11.31001
![z10-lubeck-schwerin-current](https://user-images.githubusercontent.com/42757252/66988179-0d2dac80-f0fd-11e9-9afb-c57278a57fc3.png)

z11 Schwerin
![z11-schwerin-current](https://user-images.githubusercontent.com/42757252/66988600-ca200900-f0fd-11e9-928f-d84a6a236dd6.png)

### z10 Luxembourg
https://www.openstreetmap.org/#map=12/49.6490/6.1601
![z10-luxembourg-before](https://user-images.githubusercontent.com/42757252/66988213-1dde2280-f0fd-11e9-8e59-7216419d54ac.png)

z11 Luxembourg
![z11-luxembourg-before](https://user-images.githubusercontent.com/42757252/66988231-220a4000-f0fd-11e9-9f25-398c5869985d.png)

z12 Luxembourg
![z12-n-luxembourg-before](https://user-images.githubusercontent.com/42757252/66988241-2898b780-f0fd-11e9-9496-1da7517b3200.png)

### z8 Cardiff, Wales
![z8-cardiff-faded](https://user-images.githubusercontent.com/42757252/66988281-38b09700-f0fd-11e9-9299-900b1324e47d.png)

z9 Cardiff
![z9-cardiff-faded](https://user-images.githubusercontent.com/42757252/66988296-3fd7a500-f0fd-11e9-9513-b6c58661b623.png)

z10
![z10-newport-faded](https://user-images.githubusercontent.com/42757252/66988308-45cd8600-f0fd-11e9-9d85-ac43898f5f80.png)

z11
![z11-newport-faded](https://user-images.githubusercontent.com/42757252/66988323-4c5bfd80-f0fd-11e9-8b9b-3fc8fc2a00a5.png)

z12
![z12-newport-faded](https://user-images.githubusercontent.com/42757252/66988332-52ea7500-f0fd-11e9-95c3-4f2c81faaa81.png)

**Comment by kocio-pl** [comment]
I think the problem description is not too accurate nor the problem is hitting the right spot.

The description is about roads, but there are roads which are very visible (motorroads) and much more other roads which are hardly visible or already comparable with waterways.

The problem in itself is not well observed in my opinion. "This suggests that this style is focused on roads, especially motorroads, at mid and low-mid zoom levels." I think this is skewed, since midzoom is also focused on water areas and big cities. And midzoom is a proper level for these objects to be better visible than the others.

Having said that, I think it would make more sense to discuss it after #3936 is discussed and possibly applied. Both issues are somewhat coupled together.

**Comment by jeisenbe** [comment]
> here are roads which are very visible (motorroads) and much more other roads which are hardly visible or already comparable with waterways.

Right, it's mainly motorways and trunk roads that are emphasized at z7 to z11.

But the orange primary roads are still stronger than rivers, railways and landcover at z11 to z8. And even the gray secondary highways are too strong over the faded farmland and meadow colors at z9 to z11 - though they are not stronger than railways or rivers

> discuss it after #3936

Right, fixing #3936 will mostly fix this issue, though we should also think about how waterways, railways and other linear features look compared to roads. 

A good comparison to think about is the openstreetmap.fr style, which shows landcover in stronger colors and roads less prominently at lower zoom levels.

**Comment by imagico** [comment]
Closing as per https://github.com/openstreetmap-carto/openstreetmap-carto/issues/3513#issuecomment-4229492625
